install_config/configuring_local.adoc

```diff
@@ -109,32 +81,28 @@ kind: ConfigMap
 metadata:
   name: local-volume-config
 data:
-    "local-ssd": | <1>
-      {
-        "hostDir": "/mnt/local-storage/ssd", <2>
-        "mountDir": "/mnt/local-storage/ssd" <3>
-      }
-    "local-hdd": |
-      {
-        "hostDir": "/mnt/local-storage/hdd",
-        "mountDir": "/mnt/local-storage/hdd"
-      }
+    storageClassMap: |
+        local-ssd: <1>
+            hostDir:  /mnt/local-storage/ssd <2>
+            mountDir: /mnt/local-storage/ssd <3>
+        local-hdd:
+            hostDir: /mnt/local-storage/hdd
+            mountDir: /mnt/local-storage/hdd
 ----
 <1> Name of the StorageClass.
 <2> Path to the directory on the host. It must be a subdirectory of *_/mnt/local-storage_*.
-<3> Path to the directory in the provisioner pod. We recommend using the same directory structure as used on the host.
+<3> Path to the directory in the provisioner pod. We recommend using the same directory structure as used on the host and `mountDir` can be omitted in this case.
 
 With this configuration, the provisioner creates:
 
 * One PV with StorageClass `local-ssd` for every subdirectory in *_/mnt/local-storage/ssd_*.
 * One PV with StorageClass `local-hdd` for every subdirectory in *_/mnt/local-storage/hdd_*.
```

Comment:
In openshift 3.10, local provisioner will create PV just for mountpoint.

Reply:
Fixed